src/ASM/FiniteElement.C

```diff
@@ -42,7 +48,7 @@ std::ostream& FiniteElement::write (std::ostream& os) const
 std::ostream& MxFiniteElement::write (std::ostream& os) const
 {
   this->FiniteElement::write(os);
-  for (size_t b = 0; b+1 < Nx.size(); b++)
+  for (size_t b = 0; b < Nx.size(); b++)
   {
```

Comment:
Not sure, but to me it looks like it should have been "for (size_t b = 1; b < Nx.size(); b++)"
Or else the first basis is printed twice, due to calling the parent class method?